engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByWindowTicks.java

```diff
@@ -27,48 +27,26 @@
  * of `influencer` values to add to the rolling window as the current row changes.
  */
 public class UpdateByWindowTicks extends UpdateByWindow {
-    private static final int WINDOW_POS_BUFFER_INITIAL_CAPACITY = 512;
-    protected final long prevUnits;
-    protected final long fwdUnits;
+    private final long prevUnits;
+    private final long fwdUnits;
```

Comment:
We should ensure that the window is always smaller than the maximum chunk size in our input validation.